E2E/assertions/copyLinkAssert.ts

```diff
@@ -0,0 +1,17 @@
+import assert = require("assert");
+import { displayPartsToString } from "typescript";
+import gamePage = require("../pageObjects/game.page");
+const { I } = inject();
+
+export = {
+    async checkIfTheLinkWasCopied () {
```

Comment:
Why was this moved into a separate file? Cannot we do this exact thing in the test itself?